app/config/config_dev.yml

```diff
@@ -23,6 +23,16 @@ monolog:
 #        chromephp:
 #            type:  chromephp
 #            level: info
+        console:
+          type: console
+          process_psr_3_messages: false
+          channels: ['!event', '!doctrine', '!console']
+        # To follow logs in real time, execute the following command:
```

Comment:
In all other places we add comments above the item, so I don't get why this should not follow that convention.
But no deal breaker for me.